recipes/example/meta.yaml

```diff
@@ -92,4 +92,4 @@ extra:
     # GitHub IDs for maintainers of the recipe.
     # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
     - LisaSimpson
-    - LandoCalrissian
+    - LandoCalrissian
```

Comment:
Can you revert the change in our example recipe? If that is complicated for you I can do it for you, just let me know.

Reply:
Please, you can proceed!

Reply:
Looks like you do not have a "let maintainers edit" marked on your PR. You can do that with `git co 4e6727a61e7cb82d5db2d3171ba460e908c0ead7 -- meta.yaml` and then commit it.

BTW, I'd address the recipe you are submitting first.

Reply:
Sorry, that was my mistake. I unchecked this button.

Reply:
Still cannot edited it. You'll have to fix this one yourself or open a new PR.

Reply:
I just revert the `example/meta.yaml` file. I hope now it is okay to merge this PR.